tomcat-gcp-lib/pom.xml

```diff
@@ -30,6 +30,38 @@
     </dependency>
 
     <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-all</artifactId>
```

Comment:
To settle conflicts between the version of grpc in `google-cloud-datastore` and `trace-grpc-api-service`. The trace library depends on `grpc-all` and the `cloud-datastore` only on a subset of grpc. 

`netty-tcnative-boringssl-static` is required for SSL (https://github.com/grpc/grpc-java/blob/master/SECURITY.md)